XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +341,10 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func changeBrowserView(view: String) {
```

Comment:
I'm not sure if I am doing something wrong here...
If I do: 

> navigator.changeBrowserView(view: .desktop)

I get an error: Type ‘String’ has no member ‘desktop'
The only way I can pass it is by calling it this way: 

> navigator.changeBrowserView(view: BrowserMenuChoice.desktop.rawValue)